### PR TITLE
fix bug in rm -fa parallel deletes

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: cmd/podman/ps.go [hunk 1]
```diff
@@ -344,7 +344,9 @@ func psCmd(c *cli.Context) error {
 		// Output Namespace headers
 		fmt.Fprintf(w, "\n%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s", hid, hnames, nspid, nscgroup, nsipc, nsmnt, nsnet, nspidns, nsuserns, nsuts)
 	}
-
+	if len(pss) == 0 {
+		fmt.Fprint(w, "\n")
+	}
```
Comment (TomSweeneyRedHat) [inline cmd/podman/ps.go]: I'm assuming this was an intended add on.

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: cmd/podman/ps.go (hunk 1 above)
Comment (baude) [reply cmd/podman/ps.go]: yeah, @mheon noted that there is no newline after the ps header.

Comment (TomSweeneyRedHat) [reply cmd/podman/ps.go]: cool, thanks.

**Pull request merged**